FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -50,24 +51,25 @@ FBSnapshotVerifyViewWithOptions(view__, identifier__, FBSnapshotTestCaseDefaultS
  @param layer The layer to snapshot
  @param identifier An optional identifier, used is there are multiple snapshot tests in a given -test method.
  @param suffixes An NSOrderedSet of strings for the different suffixes
+ @param tolerance The percentage of pixels that can differ and still count as an 'identical' layer
  */
-#define FBSnapshotVerifyLayerWithOptions(layer__, identifier__, suffixes__) \
+#define FBSnapshotVerifyLayerWithOptions(layer__, identifier__, suffixes__, tolerance__) \
 { \
 NSError *error__ = nil; \
 BOOL comparisonSuccess__; \
 XCTAssertTrue((suffixes__.count > 0), @"Suffixes set cannot be empty %@", suffixes__); \
 for (NSString *suffix__ in suffixes__) { \
 NSString *referenceImagesDirectory__ = [NSString stringWithFormat:@"%s%@", FB_REFERENCE_IMAGE_DIR, suffix__]; \
-comparisonSuccess__ = [self compareSnapshotOfLayer:(layer__) referenceImagesDirectory:referenceImagesDirectory__ identifier:(identifier__) error:&error__]; \
+comparisonSuccess__ = [self compareSnapshotOfLayer:(layer__) referenceImagesDirectory:referenceImagesDirectory__ identifier:(identifier__) tolerance:(tolerance__) error:&error__]; \
 if (comparisonSuccess__ || self.recordMode) break; \
 } \
 XCTAssertTrue(comparisonSuccess__, @"Snapshot comparison failed: %@", error__); \
 XCTAssertFalse(self.recordMode, @"Test ran in record mode. Reference image is now saved. Disable record mode to perform an actual snapshot comparison!"); \
 }
 
-#define FBSnapshotVerifyLayer(layer__, identifier__) \
+#define FBSnapshotVerifyLayer(layer__, identifier__, tolerance__) \
```

Comment:
Right, we should keep this the same and only append a parameter on `FBSnapshotVerifyLayerWithOptions` (like is done already)